compiler/one-cmds/tests/rawdata2hdf5_neg_004.test

```diff
@@ -19,7 +19,7 @@ filename="${filename_ext%.*}"
 
 trap_err_onexit()
 {
-  if grep -q "Unable to create file" "${filename}.log"; then
```

Comment:
Error message has changed.

```
FileNotFoundError: [Errno 2] Unable to synchronously create file
```

"synchronously" is newly added. Instead of detailed message, let's use Error name (FileNotFoundError) for testing.